runtime/src/asset.rs

```diff
@@ -505,7 +548,7 @@ decl_module! {
             ensure!(<identity::Module<T>>::is_signer_authorized(did, &signer), "sender must be a signing key for DID");
             ticker.canonize();
             ensure!(<BalanceOf<T>>::exists((ticker, did)), "Account does not own this token");
-
+            ensure!(!Self::frozen(&ticker), "token is frozen");
```

Comment:
I think we don't need this check here.. `transferFrom` will fail automatically because of the frozen check. We can allow `approve` function as an allowance can be used later once transaction gets unfreeze

Reply:
There are no iterations when the vectors are empty. `batch_issue` succeeds vacuously.